src/styles/main.scss

```diff
@@ -1,3 +1 @@
-body {
-  margin: 0;
-}
+@import "header", "navigation", "container", "card", "reset";
```

Comment:
`@import` is outdated, you should better use `@use`.
Also separate imports of utils files (make sure you put them at the top) and blocks